README.md

```diff
@@ -0,0 +1,20 @@
+# Release Information
+
+- **Version**: 1.0.0
+
+- **Certified**: Yes
+
+- **Publisher**: Fortinet  
+
+- **Compatibility**: 7.3.1 and later
```

Comment:
@cs-abhishek-shukla : Please change the FSR compatibility as 7.4.3 & later

Reply:
Done